test/rpc/test_deps.py

```diff
@@ -72,42 +72,80 @@ def deps_with_packages(packages, bad_packages, project_dir, profiles_dir, schema
         querier.is_result(querier.async_wait(tok1))
 
 
+@pytest.mark.parametrize(
+    "packages, bad_packages",
+    # from dbt hub
+    [(
+        [{
+            'package': 'fishtown-analytics/dbt_utils',
+            'version': '0.5.0',
+        }],
+        # wrong package name
+        [{
+            'package': 'fishtown-analytics/dbt_util',
+            'version': '0.5.0',
+        }],
+    ),
+    # from git release/tag/branch
+    (
+        [{
+            'git': 'https://github.com/fishtown-analytics/dbt-utils.git',
+            'revision': '0.5.0',
+        }],
+        # if you use a bad URL, git thinks it's a private repo and prompts for auth
+        [{
+            'git': 'https://github.com/fishtown-analytics/dbt-utils.git',
+            'revision': 'not-a-real-revision',
+        }],
+    ),
+    # from git commit
+    (
+        [{
+            'git': 'https://github.com/fishtown-analytics/dbt-utils.git',
+            'revision': 'b736cf6acdbf80d2de69b511a51c8d7fe214ee79',
+        }],
+        # don't use short commits
+        [{
+            'git': 'https://github.com/fishtown-analytics/dbt-utils.git',
+            'revision': 'b736cf6',
+        }],
+    ),
+    # from git release and subdirectory
+    (
+        # TODO: Change me to something that fishtown-analytics manages!
+        # here I just moved the dbt_utils code into a subdirectory
+        [{
+            'git': 'https://github.com/dmateusp/dbt-utils.git',
+            'revision': 'dmateusp/0.5.0/move_dbt_utils_to_subdir',
+            'subdirectory': 'dbt_projects/dbt_utils',
+        }],
+        [{
+            'git': 'https://github.com/fishtown-analytics/dbt-utils.git',
+            'revision': '0.5.0',
+            'subdirectory': 'path/to/nonexistent/dir',
+        }],
+    ),
```

Comment:
```suggestion
        [{
            'git': 'https://github.com/fishtown-analytics/dbt-labs-experimental-features.git',
            'revision': '0.0.1',
            'subdirectory': 'materialized-views',
        }],
        [{
            'git': 'https://github.com/fishtown-analytics/dbt-labs-experimental-features.git',
            'revision': '0.0.1',
            'subdirectory': 'path/to/nonexistent/dir',
        }],
    ),
```

this should do the trick!

Reply:
Done! thanks!